src/NUnitConsole/nunit3-console.tests/CommandLineTests.cs

```diff
@@ -522,6 +523,18 @@ public void ShouldSetTeamCityFlagAccordingToArgsAndDefauls(bool hasTeamcityInCmd
 
         #endregion
 
+        #region Testlist Option
+
+        [Test]
+        public void ShouldNotFailOnEmptyLine()
+        {
+            var testListPath = Path.Combine(TestContext.CurrentContext.TestDirectory, "TestListWithEmptyLine.tst");
+            var options = new ConsoleOptions("--testlist=" + testListPath);
+            Assert.That(options.errorMessages, Is.Empty);
+        }
```

Comment:
Agreed - if I was writing this, I'd want to ensure that empty strings were definitely being excluded here and not passed through to the test list, in case of future changes here.